onnxruntime/core/providers/cpu/tensor/col2im_attributes.h

```diff
@@ -0,0 +1,50 @@
+/**
+* Copyright (c) 2016-present, Facebook, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+/* Modifications Copyright (c) Microsoft. */
+
+#pragma once
+
+#ifndef SHARED_PROVIDER
```

Comment:
why do we need the SHARED_PROVIDER ifdef here?

Reply:
Probably copied from transpose.h and never removed it. If it is not needed, will remove it